test/verifier_test.js

```diff
@@ -634,16 +634,20 @@ describe('Verifier tests', function() {
 
     return new Promise(async function(resolve) {
       let a = await initStuffForTaskCreation();
-      let task = new DeployTask(a.secretContractAddress, a.preCode, a.encryptedArgs, a.encryptedFn, a.userDHKey, a.gasLimit, a.secretContractAddress);
+      let blockNumber = a.expectedParams.firstBlockNumber + 50;
+
+      let task = new DeployTask(a.secretContractAddress, a.preCode, a.encryptedArgs, a.encryptedFn, a.userDHKey,
+        a.gasLimit, a.secretContractAddress, 0);
```

Comment:
It might be clearer if you put a const `BLOCK_NUMBER=0` or something in that sort. It seems to come up a lot in this file.

Reply:
Where does the `+50` come from?

Reply:
The [EnigmaContractMock](https://github.com/enigmampc/enigma-p2p/blob/master/test/ethereum/EnigmaContractMock.js) used in the test is being initialized with epochs in the size of [100](https://github.com/enigmampc/enigma-p2p/blob/master/test/ethereum/utils.js#L93) blocks so the number 50 represents the middle of an epoch.